### ISSUE TITLE
Show motorway links from zoom 11+

### ISSUE THREAD

**Issue by zerebubuth** [mapzen/vector-datasource#488]
From #358:

> We should stop including motorway_link at zooms 8, 9, and 10.

**Comment by nvkelso** [comment]
tl;dr

We're good. There are still a sprinkling of other data-striping issues (see screenshots below), but the `motorway_link` problem described in this issue has been resolved. 

===

I loaded up the [scene.txt](https://github.com/mapzen/vector-datasource/files/23796/scene.txt) file (rename txt to yaml) in Tangram Play to get these before / after images:

**new new (dev):**
<img width="1220" alt="screen shot 2016-01-20 at 15 33 36" src="https://cloud.githubusercontent.com/assets/853051/12466920/ae6d9f5c-bf8c-11e5-9e7e-6c8f6a786532.png">


**new (dev):**
![screen shot 2016-01-14 at 16 09 42](https://cloud.githubusercontent.com/assets/853051/12342197/119b565a-badd-11e5-9e7b-239931bcca9c.png)

**old (prod):**
![screen shot 2016-01-14 at 16 09 36](https://cloud.githubusercontent.com/assets/853051/12342198/119d63dc-badd-11e5-9c0f-4734edf844be.png)

It looks like the remaining sprinkling of tiny road segments are `motorway_link` features in the `kind: highway` grouping:
